app/src/main/java/com/example/mediadb/view/movielist/MovieListViewModel.kt

```diff
@@ -41,4 +63,8 @@ class MovieListViewModel constructor(val movieRepository: MovieRepository) : Bas
     fun showListMovieData(): MutableLiveData<MutableList<Movie>> {
         return listMovieData
     }
+
+    fun showListFavoriteMovie(): MutableLiveData<MutableList<Movie>> {
```

Comment:
Tại sao anh không dùng thẳng thằng listFavoriteMovie, bản thân mỗi variable nó đều có phương thức get set mặc định cả, khi anh gọi thằng listFavoriteMovie thì nó sẽ ngầm chạy vào get set mặc định của variable đó

Reply:
bởi vì 2 biến này a cần phải để private nên sẽ không truy cập trực tiếp vào biến đấy đc.
Còn nếu không để private thì ide sẽ notify: Inspection info: This inspection reports class members which can be made private.

Reply:
Tại sao anh phải để private, anh dùng ở đâu đó thì nó sẽ không warning cái này nữa, nó warning là do khi anh khởi tạo anh chưa dùng ở đâu cả